Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.10
+
+RUN apt-get update && apt-get install -y \
+    build-essential                      \
+    python3.6                            \
```

Comment:
I'm confused about this package. I'm pretty sure this does not exist in Ubuntu 22.10, only 18.04: https://packages.ubuntu.com/search?keywords=python3.6. Also, when I tried building the docker image, the version of python that I got was 3.10.

Reply:
Agreed, replaced with `python3` in 647d63600d667e1d561a217235504667df1d513f